software/control/core/core.py

```diff
@@ -3785,122 +2285,6 @@ def scale_contrast_limits(self, target_dtype):
         self.acquisition_dtype = target_dtype
 
 
-class PlateReaderNavigationController(QObject):             # Not implemented for Prior stage
```

Comment:
This was unused.  `ag PlateReaderNavigationController` only showed this occurence.

Reply:
Yes this obsolete and can be deleted.